PixaPicker/PixaDataCoordinatorDelegate.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  PixaDataCoordinatorDelegate.swift
+//  PixaPicker
+//
+//  Created by Thomas Caraway on 11/22/19.
+//  Copyright © 2019 Thomas Caraway. All rights reserved.
+//
```

Comment:
There's no problem making this its own file, but you can usually get away with including the delegate protocol in the same file with the class it is delegating

Reply:
WHAT?! This is good to know. Reminds me of C.